CondFormats/Common/src/SmallWORMDict.cc

```diff
@@ -1,7 +1,10 @@
-#include "CondFormats/Common/interface/SmallWORMDict.h"
+#include <algorithm>
 #include <string>
 #include <functional>
 #include <numeric>
+#include <vector>
```

Comment:
This is already included in the header, not needed to be repeated here:
```suggestion
```

Reply:
> I can make the change, though I'm not fond of that approach, because if one removes the `#include` from the header then they have to update also the `.cc` file.

Thank you @fwyzard 
Well, in this case both `vector` and `string` are only used as arguments of the class methods (or related to it): if the corresponding `#includes`'s are removed from the header, it means that those methods will become completely different, and the .cc file would necessarily need to be modified in any case.